packages/doenetml-prototype/src/renderers/components/section.tsx

```diff
@@ -3,15 +3,19 @@ import { BasicComponentWithPassthroughChildren } from "../types";
 import { Element } from "../element";
 
 export const Section: BasicComponentWithPassthroughChildren<{
-    titleElmId?: number;
+    props: { title?: number; codeNumber: string };
 }> = ({ children, node, visibilityRef }) => {
-    const titleElmId = node.data.titleElmId;
+    const titleElmId = node.data.props.title;
+    const codeNumber = node.data.props.codeNumber;
+    const displayName = `Section${codeNumber ? ` ${codeNumber}.` : ""}`;
 
-    const title = titleElmId != null ? <Element id={titleElmId} /> : "Section";
+    const title = titleElmId != null ? <Element id={titleElmId} /> : "";
 
     return (
         <div className="section" ref={visibilityRef}>
-            <h3>{title}</h3>
+            <h3>
+                {displayName} {title}
```

Comment:
Eventually, we want the ability to suppress `displayName` or just the word `Section`. Not that we need to add that yet, but just noting that will be a future requirement. (Also adjusting the length of the look-back in `codeNumber`.)

(Since DoenetML is typically used in small activities, I was asked to suppress `displayName` by default.)

Reply:
Yep! This is just the workaround for the moment. Things will get more detailed when we have all the props implemented.